tests/include/WriteCountingBackend.h

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: Deutsches Elektronen-Synchrotron DESY, MSK, ChimeraTK Project <chimeratk-support@desy.de>
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
```

Comment:
I think this fails the code style check without `#pragma once`

Reply:
good point, thanks